.github/workflows/npm-cd.yml

```diff
@@ -69,12 +74,53 @@ jobs:
                           arch: arm64,
                           TARGET: aarch64-apple-darwin,
                       }
+                    - {
```

Comment:
can you remove the `CONTAINER: "2_28",` from line 61? it's a leftover from pypi-cd code